oauth2_provider/tests/test_authorization.py

```diff
@@ -0,0 +1,107 @@
+"""
+Tests various flows through using the Authorization Endpoint.
+"""
+from urlparse import urlparse, parse_qs
+
+import ddt
+import provider.oauth2.models
+import provider.scope
+
+from oauth2_provider.tests.base import OAuth2TestCase, IDTokenTestCase
+
+
+@ddt.ddt
+class ImplicitFlowTestCase(IDTokenTestCase):
+    """
+    Test OIDC-related extensions to the OAuth2 Authorization flows
+    implemented by this package.
+    """
+
+    def _get_auth_payload(self, location):
+        """
+        DRY helper.  Extract auth parameters from the redirect location as
+        returned during the implicit flow.
+        """
+        self.assertTrue(location.startswith(self.auth_client.redirect_uri + '#'))
+        return {k: v[0] for k, v in parse_qs(urlparse(location).fragment).items()}
+
+    def _get_access_token(self, scope):
+        """
+        DRY helper.  Find the access token generated during our auth request.
+        """
+        return provider.oauth2.models.AccessToken.objects.get(
```

Comment:
I would import `AccessToken` from `provider.oauth2.models` above and change this to 
```python
return AccessToken.objects.get(...)
```